core/tee/tee_svc_storage.c

```diff
@@ -932,9 +932,9 @@ TEE_Result syscall_storage_obj_trunc(unsigned long obj, size_t len)
 			if (res != TEE_SUCCESS)
 				goto exit;
 			res = TEE_ERROR_CORRUPT_OBJECT;
-			goto exit;
 		} else
 			res = TEE_ERROR_GENERIC;
```

Comment:
recommend to use braces around this block (since it is an else of a braced block).